Operating System/pagereplacementalgorithm.md

```diff
@@ -0,0 +1,159 @@
+# Page Replacement Algorithm: Least Recently Used (LRU)

```

Comment:
Please follow the syntax and format for our docs. If you have any questions, please don't hesitate to ask.